.changeset/strong-toes-sniff.md

```diff
@@ -0,0 +1,6 @@
+---
+'@aws-amplify/backend-data': minor
+'@aws-amplify/backend': minor
+---
```

Comment:
```suggestion
---
'@aws-amplify/backend-data': minor
---
```

Reply:
that won't pass checks, see https://github.com/aws-amplify/amplify-backend/pull/2042 .

Reply:
So this is still an issue?

Reply:
Not an issue.
We enforce that `backend` has to inherit version bump from `backend-X` packages. We may revisit if that check makes sense, but that's why we have both in this PR.